app/assets/javascripts/app/datepicker-data.coffee

```diff
@@ -3,12 +3,16 @@ class window.DatePickerData
     # This will set up a datepicker based on data attributes
     # Make sure you call `to_s` or `iso8601` on any dates you set in the views
     # to ensure the proper format.
-    $picker = $(".datepicker__data")
+    $pickers = $(".datepicker__data")
 
-    $picker.datepicker(
-      minDate: new Date($picker.data("min-date")) # will be unbounded if not provided
-      maxDate: new Date($picker.data("max-date")) # will be unbounded if not provided
-    )
+    $pickers.each (i, picker) ->
+      $picker = $(picker)
+      $picker.datepicker(
+        minDate: new Date($picker.data("min-date")) # will be unbounded if not provided
+        maxDate: new Date($picker.data("max-date")) # will be unbounded if not provided
+      )
+
+    new DatePickerValidate($pickers).activate()
```

Comment:
This felt like a reasonable initialization process. I could possibly be convinced to squash these two classes together.